guides/plugins/plugins/storefront/add-cookie-to-manager.md

```diff
@@ -58,6 +66,9 @@ Let's have a look at an example:
 namespace PluginName\Framework\Cookie;
 
 use Shopware\Storefront\Framework\Cookie\CookieProviderInterface;
+use Shopware\Core\Content\Cookie\Struct\CookieEntry;
+use Shopware\Core\Content\Cookie\Struct\CookieGroup;
+use Shopware\Core\Framework\Struct\Collection;
```

Comment:
nope, this does not work like that now :see_no_evil: 

see https://github.com/shopware/shopware/blob/trunk/src/Core/Framework/App/Cookie/AppCookieCollectListener.php